File: dotnet/windntrees.core/DataAccess.Core/Models/Configuration.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace DataAccess.Core.Models
{
    public partial class Configuration
    {
        [Key]
        [Column("UID")]
        public Guid Uid { get; set; }
        [Required]
        [StringLength(50)]
        public string KeyParam { get; set; }
        public byte[] RowVersion { get; set; }
        [Required]
        [StringLength(1500)]
        public string ValParam { get; set; }
    }
}

File: dotnet/windntrees.core/Application.Core/wwwroot/js/models/Configuration.js
function Configuration(data) {
    var instance = this;
    instance._datakey = data.Uid;

    instance.Uid = data.Uid;
    instance.KeyParam = ko.observable(data.KeyParam);
    instance.ValParam = ko.observable(data.ValParam);
    instance.RowVersion = data.RowVersion;
}